Sources/App/Models/Version.swift

```diff
@@ -132,6 +132,14 @@ final class Version: Model, Content {
         case defaultBranch = "default_branch"
         case preRelease = "pre_release"
         case release
+
+        var cssClass: String {
+            switch self {
+                case .preRelease: return "beta"
+                case .defaultBranch: return "branch"
+                case .release: return "stable"
+            }
+        }
```

Comment:
Nit: View-related stuff should probably live in a little extension near the view code under `App/Views` somewhere rather than in the model file itself.

Reply:
Ah, I meant to comment this myself when I opened the PR. This conversion is needed from two places, and didn't feel like it deserved a whole file just for this.

Reply:
I'd still move it out of the models though and keep any view related code in that hierarchy 🤷

Reply:
I moved it. There was no sensible place for these to live under Views so I made a new ModelExtensions directory.